scania-digital-design-system/sdds
# fix: merging main_4.10.0 code to main

**Describe pull-request**  
Merging code from main_4.10.0 components folder and theme-light/_core.scss.

**Solving issue**  
Fixes: [AB#2498](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2498)

**How to test**  
1. Clone sdds once again
2. In one repo run this branch, in another one main_4.10.0 branch
3. In both, `run npm` start from root level
4. make sure that Storybook looks exactly the same in both of them.
5. @EmelieLitwin , @AlbinFarstun and @mJarsater - please check work you did for main_4.10.0 is found in this feature branch - to remind you, I have added a screenshot of changes below
6. table component and icon component has to be the same as in [production Storybook](https://production.d1g8v8mrkx992n.amplifyapp.com/?path=/story/foundation-colour--brand)

**Additional context**  
This one is done in a simple manner, I had two copies of sdds repo on my local. I have copy pasted components package from main_4.10.0 branch of one repo in a feature branch of new repo. What is also copied is core.scss from theme-light as Martin worked on small change there.  
After that I deleted node_modules and package-lock.json file in both component and theme/light packages and run fresh installation to be one the "safe" side.
I run the project with `npm start` after it and it loads OK.
I had to commit with "--no-verify" flag as old code has many errors in linting now due to changes we did with Eslint.

**Changes done in main_4.10.0 branch **

![Screenshot 2022-11-14 at 10 12 34](https://user-images.githubusercontent.com/17004063/201714203-123efd88-8095-4a01-800c-611d2c58ec59.png)
